### PR TITLE
Compare it

### PR DESCRIPTION
**Pull request by eurecho**
Basta compare

### CODE REVIEW

**Comment by AljayAgustin** [comment]
on it

**Comment by AljayAgustin** [comment]
> Basta compare

okay

**Review by AljayAgustin — APPROVED** [review]

File: SortingAlgorithms.java
Comment (grgn2002) [inline SortingAlgorithms.java:1]: Add more sorthing methods.

**Review by grgn2002 — CHANGES_REQUESTED** [review]
Request can be read on the comments.

**Pull request merged**